pctasks/core/pctasks/core/models/task.py

```diff
@@ -83,9 +83,7 @@ class TaskRunMessage(PCBaseModel):
     config: TaskRunConfig
 
     def encoded(self) -> str:
-        return b64encode(
-            self.json(exclude_unset=True, exclude_none=True).encode("utf-8")
```

Comment:
This `exclude_unset` was the culprit for not serializing `created` or other non-initialized fields. It didn't look like that behavior was intentional, but let me know if there was a specific reason.

Reply:
Yeah, it's a weird settings. From the docs:

> exclude_unset: whether fields which were not explicitly set when creating the model should be excluded from the returned dictionary; default False. Prior to v1.0, exclude_unset was known as skip_defaults; use of skip_defaults is now deprecated